tests/plugins/os/unix/bsd/darwin/ios/test__os.py

```diff
@@ -0,0 +1,46 @@
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.plugin import OperatingSystem
+from dissect.target.plugins.os.unix.bsd.darwin.ios._os import IOSPlugin
+from dissect.target.plugins.os.unix.bsd.darwin.ios.applications import (
+    IOSApplicationsPlugin,
+)
+from dissect.target.plugins.os.unix.bsd.darwin.ios.locale import LocalePlugin
+from dissect.target.target import Target
+from tests._utils import absolute_path
+
+
+def test_unix_bsd_darwin_ios_os(target_ios: Target, fs_ios: VirtualFilesystem) -> None:
+    """Test if we correctly detect an iOS target.
```

Comment:
It doesn't test detection since you manually add the `IOSPlugin`, only tests some plugin functionality now.